### PR TITLE
Create Android lib, add Podfile. 

### PR DESCRIPTION
**Pull request by designatednerd**

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/ios-developers For you: Podfile and ReadMe. 
@vokal/android-developers For you: I updated things so that they're all in a library now, and updated some of the instructions.

File: ParchePartnerURLScheme.podspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name         = "ParchePartnerURLScheme"
+  s.version      = '1.0.0'
+  s.summary      = "Parche Partner URL Scheme Helper"
+  s.description  = <<-DESC
+                    A utility to make it easier for partner applications of Parche to open the app using a URL scheme.
```
Comment (seanwolter) [inline ParchePartnerURLScheme.podspec]: what is this line with the `<<-DESC` ? How podspec work?

Comment (designatednerd) [inline ParchePartnerURLScheme.podspec]: My recollection is that it lets you use whitespace within your description.

**Comment by DavidRyan** [comment]
lgtm

File: ParchePartnerURLScheme.podspec (hunk 1 above)
Comment (vokal-isaac) [inline ParchePartnerURLScheme.podspec]: It's [Ruby's here-document syntax](http://en.wikipedia.org/wiki/Here_document#Ruby) for a string literal.  I'd say it's more about being able to use quotes and whatever else without having to escape them plus simple multiline abilities.

File: ParchePartnerURLScheme.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                    
+                    More information
```
Comment (vokal-isaac) [inline ParchePartnerURLScheme.podspec]: Is this trailing "More information" intentional?

**Comment by vokal-isaac** [comment]
iOS bits (podspec and readme) LGTM, other than my one question.

File: ParchePartnerURLScheme.podspec (hunk 1 above)
Comment (scottferg) [inline ParchePartnerURLScheme.podspec]: Ruby strikes again!

File: ParchePartnerURLScheme.podspec (hunk 2 above)
Comment (designatednerd) [inline ParchePartnerURLScheme.podspec]: Whoops, no it is not. Good catch.

**Pull request merged**